chamber/storages/boto3.py

```diff
@@ -0,0 +1,28 @@
+from django.core.files.base import ContentFile
+
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+__all__ = (
+    'S3Storage',
+    'force_bytes_content',
+)
+
+
+def force_bytes_content(content):
+    _content = content.read()
```

Comment:
nedá se přečíst jen pár prvních bytů? Načítat celý soubor kvůli tomu abys zjistil, že je to byte array je hloupé ne?

Reply:
Pravda... měl by dokonce stačit opravdu první pár (dva).